build.sh

```diff
@@ -102,11 +102,11 @@ function all_frameworks_test() {
 function strip_framework() {
   local FRAMEWORK_PATH="$BUILD_DIRECTORY/Build/Products/Debug/$1"
   echo "Stripping Framework $FRAMEWORK_PATH"
-  rm -r "$FRAMEWORK_PATH"
+  rm -rf "$FRAMEWORK_PATH"
```

Comment:
I've added it so it does not fail (due to `set -e`) if you try to remove something that does not exist. This way `cli_build` can be shared between `fbsimctl` and `fbxctest`. Otherwise we have to provide a separate one for each of them, because in `fbxctest` you won't have to strip anything in `FBSimulatorControlKit.framework` because that's just not being built for that target.
If we still want to share `cli_build`, but dislike `-f`, I can make the `strip_framework` check for existance of given path, before removing it.
What do you think?